gensim/topic_coherence/text_analysis.py

```diff
@@ -345,14 +360,14 @@ def queue_all_texts(self, q, texts, window_size):
         """Sequentially place batches of texts on the given queue until `texts` is consumed.
         The texts are filtered so that only those with at least one relevant token are queued.
         """
-        relevant_texts = (text for text in texts if self.text_is_relevant(text))
-        for batch_num, batch in enumerate(self.yield_batches(relevant_texts)):
+        for batch_num, batch in enumerate(self.yield_batches(texts)):
             q.put(batch, block=True)
             before = self._num_docs / self.log_every
             self._num_docs += sum(len(doc) - window_size + 1 for doc in batch)
             if before < (self._num_docs / self.log_every):
-                logger.info("submitted %d batches to accumulate stats from %d documents (%d virtual)" % (
-                    batch_num, (batch_num + 1) * self.batch_size, self._num_docs))
+                logger.info("%d batches submitted to accumulate stats from %d documents (%d "
```

Comment:
I'm not sure I understand the desired format here. The hanging indent makes sense when using the `%` operator to format a string. However, my desire here is to use the [lazy formatting of the `logging` module instead](https://stackoverflow.com/a/4149231/2486302). In this case, the string and all following format arguments are equivalent arguments of the call, so [pep8 dictates alignment with the opening delimiter](https://www.python.org/dev/peps/pep-0008/#indentation).